packages/async-flow/CHANGELOG.md

```diff
@@ -1 +1,21 @@
+# Change Log
+
+All notable changes to this project will be documented in this file.
+See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
+
+### 0.1.1-u17.0 (2024-09-17)
+
+
+### Features
+
+* **async-flow:** asyncFlow ([#9097](https://github.com/Agoric/agoric-sdk/issues/9097)) ([16095c5](https://github.com/Agoric/agoric-sdk/commit/16095c5076043133aff0f25721131be2ca1ef5af)), closes [#9302](https://github.com/Agoric/agoric-sdk/issues/9302) [#9125](https://github.com/Agoric/agoric-sdk/issues/9125) [#9126](https://github.com/Agoric/agoric-sdk/issues/9126) [#9153](https://github.com/Agoric/agoric-sdk/issues/9153) [#9154](https://github.com/Agoric/agoric-sdk/issues/9154) [#9280](https://github.com/Agoric/agoric-sdk/issues/9280) [#9126](https://github.com/Agoric/agoric-sdk/issues/9126)
+* **async-flow:** endowments ([#9566](https://github.com/Agoric/agoric-sdk/issues/9566)) ([4390d8c](https://github.com/Agoric/agoric-sdk/commit/4390d8c21fd8ac80a9c83f55d38c52e3f98faa1e)), closes [Error#1](https://github.com/Agoric/Error/issues/1) [Error#1](https://github.com/Agoric/Error/issues/1)
+* **async-flow:** error on guest E use ([#9443](https://github.com/Agoric/agoric-sdk/issues/9443)) ([e193e66](https://github.com/Agoric/agoric-sdk/commit/e193e66fc578b1c00eda5f5ab6599f1cfbdc542f)), closes [#9299](https://github.com/Agoric/agoric-sdk/issues/9299) [#9322](https://github.com/Agoric/agoric-sdk/issues/9322) [#9322](https://github.com/Agoric/agoric-sdk/issues/9322) [#9322](https://github.com/Agoric/agoric-sdk/issues/9322) [#9322](https://github.com/Agoric/agoric-sdk/issues/9322) [#9322](https://github.com/Agoric/agoric-sdk/issues/9322)
+* **asyncFlow:** Stopgap E support ([#9519](https://github.com/Agoric/agoric-sdk/issues/9519)) ([4adf64f](https://github.com/Agoric/agoric-sdk/commit/4adf64fd53a1a3c68ca52728710830201c9a4418)), closes [#9322](https://github.com/Agoric/agoric-sdk/issues/9322) [#9299](https://github.com/Agoric/agoric-sdk/issues/9299) [#9443](https://github.com/Agoric/agoric-sdk/issues/9443) [#9322](https://github.com/Agoric/agoric-sdk/issues/9322) [#9322](https://github.com/Agoric/agoric-sdk/issues/9322) [#9299](https://github.com/Agoric/agoric-sdk/issues/9299) [#9322](https://github.com/Agoric/agoric-sdk/issues/9322) [#9322](https://github.com/Agoric/agoric-sdk/issues/9322) [#9322](https://github.com/Agoric/agoric-sdk/issues/9322) [#9322](https://github.com/Agoric/agoric-sdk/issues/9322)
+* error on membrane promise ([94dfab1](https://github.com/Agoric/agoric-sdk/commit/94dfab1cfd960aada452753b1482ed797bc2da13))
+* **types:** HostOf, GuestOf ([35380af](https://github.com/Agoric/agoric-sdk/commit/35380af4bbda51be2a9cd047f9c4992791090e94))
+* **types:** infer orchestrate() return ([67665a8](https://github.com/Agoric/agoric-sdk/commit/67665a89cb162c350389e52fd812c895d42f37fb))
+
+
+
 # Change Log
```

Comment:
Nit: somehow lerna decide to ignore this previous header. Maybe this file shouldn't have existed before? Not worth respinning for that however